code/modules/mob/living/simple_animal/hostile/glyphid.dm

```diff
@@ -0,0 +1,546 @@
+//Glyphids, based off the game Deep Rock Galactic
+
+#define GLYPHID_IDLE 0
+#define SPINNING_WEB 1
+#define BUILDING_TOWER 3
+#define MOVING_TO_TARGET 3
+
+/mob/living/simple_animal/hostile/glyphid
+	name = "glyphid"
+	desc = "A colony-based insectoid.  Watch those jaws."
+	icon_state = "nurse"
+	icon_living = "nurse"
+	icon_dead = "nurse_dead"
+	mob_biotypes = list(MOB_ORGANIC, MOB_BUG)
+	speak_emote = list("chitters", "roars")
+	turns_per_move = 5
+	see_in_dark = 10
+	response_help  = "rubs the hard chitin of"
+	response_disarm = "gently pushes aside"
+	response_harm   = "bashes"
+	maxHealth = 90
+	health = 90
+	speed = 0
+	var/busy = GLYPHID_IDLE
+	spacewalk = FALSE
+	ventcrawler = VENTCRAWLER_ALWAYS
+	var/playable_glyphid = TRUE
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+	deathmessage = "screeches before falling limp."
+	var/datum/action/innate/glyphid/build_tower/build_tower
+	
+	var/amount_grown = 0
+	var/max_grown = 200
+
+	obj_damage = 30
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	attacktext = "chomps on and slices"
+	attack_sound = 'sound/weapons/bite.ogg'
+	friendly = "glyphid nuzzles"
+
+	//Glyphids aren't affected by cold.
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = 1500
+
+	faction = list("glyphid")
+	gold_core_spawnable = NO_SPAWN
+	
+	do_footstep = TRUE
+	
+/mob/living/simple_animal/hostile/glyphid/Topic(href, href_list)
+	if(href_list["activate"])
+		var/mob/dead/observer/ghost = usr
+		if(istype(ghost) && playable_glyphid)
+			humanize_glyphid(ghost)
+
+/mob/living/simple_animal/hostile/glyphid/Login()
+	..()
+	to_chat(src, "<span class='notice'>Expand and defend the swarm!</span>")
+
+/mob/living/simple_animal/hostile/glyphid/attack_ghost(mob/user)
+	. = ..()
+	if(.)
+		return
+	humanize_glyphid(user)
+
+/mob/living/simple_animal/hostile/glyphid/proc/humanize_glyphid(mob/user)
+	if(key || !playable_glyphid || stat)//Someone is in it, it's dead, or the fun police are shutting it down
+		return 0
+	var/glyphid_ask = alert("Become a glyphid?", "Do you hate dwarves?", "Yes", "No")
+	if(glyphid_ask == "No" || !src || QDELETED(src))
+		return 1
```

Comment:
use TRUE/FALSE instead of 1/0 for binary returns

Reply:
Resolved.